bin/plugin/commands/release.js

```diff
@@ -448,6 +456,19 @@ async function runGithubReleaseStep(
 		log( '>> The plugin ZIP has been successfully uploaded.' );
 	} );
 
+	// Remove draft status from the Github release
+	await runStep( 'Publishing the Github release', abortMessage, async () => {
```

Comment:
Is the new step necessary? I mean not just put this logic inside the previous step. (it's all related)

Reply:
I was considering that. but since release (now draft) creation and asset upload are separate, it seemed like removing the draft status should also be separate, since it's semantically not part of the asset upload step. (I don't feel strongly about this.)